### ISSUE TITLE
[Feature request] Copy block button

### ISSUE THREAD

**Issue by j-peeters** [heyman/heynote#82]
Would be great to have a small copy block button, maybe in the top right corner of the block to copy the whole block without having to select it. The screenshot is from another app but to show the idea. 

![CleanShot 2023-12-27 at 12 54 40@2x](https://github.com/heyman/heynote/assets/4502360/ac29f54f-1872-4db2-9e27-f3953dd37b94)

**Comment by heyman** [comment]
Maybe. I'd like to keep Heynote clean and focused. There are a bunch of different useful UI elements that could be added, but if I were to add all of them, the UI would become cluttered. It could perhaps be solved by only showing the copy button when hovering close to the top right corner of the block, but it could still potentially interfere with placing the cursor in the first line (if the line is long).

**Comment by j-peeters** [comment]
Hi @heyman I understand and it's a good idea, my thought was that when people put stuff in Heynote they would like an easy way to get it out of Heynote like people asking for export functions, it falls in that catergory. 
Could it be an idea to add a shortcut to copy a block?

**Comment by heyman** [comment]
You can always do `Ctrl-A` + `Ctrl-C` to copy the content of the current block. Is a special shortcut to save an extra keystroke needed? But if so, I think it could be added as a custom command that can be manually configured once Heynote has proper custom keybindings support.

**Comment by j-peeters** [comment]
Yeah that's true, I guess maybe something like cmd-shift-copy to copy without selecting would be more in the flow than two keystrokes. But maybe other users can chime in on the request. No rush. And thanks for sharing your app with the world!

**Comment by weeheavy** [comment]
I'm already quite used to Cmd+A and it matches my rather keyboard-focused workflow perfectly. I especially like that heynote is really tidy. But I can understand that your workflow might be different.
